src/main/java/com/manardenza/entity/Hotel.java

```diff
@@ -2,10 +2,11 @@
 
 import lombok.Data;
 
+import java.io.Serializable;
 import java.util.List;
 
 @Data
-public class Hotel extends AbstractObject {
+public class Hotel extends AbstractObject implements Serializable {
```

Comment:
If you have marked AbstractObject as Serializable, this class will become so as well